docs/manual/docs/cmd/yammer/network/network-list.md

```diff
@@ -13,11 +13,17 @@ yammer network list [options]
 Option|Description
 ------|-----------
 `--help`|output usage information
-`--includeSuspended true|false`|Include the networks the user is suspended. Default `false`
+`--includeSuspended`|Include the networks the user is suspended.
```

Comment:
This sounds a bit odd. Could we please rephrase it to make it clearer what it actually means? Are you trying to say _Include the networks in which the user is suspended_?

Reply:
@waldekmastykarz: I rephrased the documentation a little. Let me know if this is fine for you now.